doc/autoscale/experiment/autoscale_experiment.md

```diff
@@ -60,4 +60,37 @@ metrics | QPS(1w) | QPS(10w) | QPS(50w)
 -- | -- | -- | --
 Trainer Pods | 100 | 80 | 50
 Nginx Pods | 80 | 100 | 150
-CPU utils| 100% | 100% | 100%
\ No newline at end of file
+CPU utils| 100% | 100% | 100%
+
+## Reproduce the experiment
+
+- Configure kubectl on your host
+- Submit the TrainingJob controller with YAML file
+    ```bash
+    > git clone https://github.com/PaddlePaddle/cloud.git && cd cloud
+    > kubectl create -f k8s/controller/trainingjob_resource.yaml
+    > kubectl create -f k8s/controller/controller.yaml
+    ```
+- Test Case1
+    1. Run the data collecting Python program.
+        ```bash
+        > cd cloud/doc/autoscale/experiment/python
+        > python main.py case1 mnist1,mnist2
+        ```
+    1. Submit two general jobs naming mnist1 and mnist2 as following,
```

Comment:
Is it possible that you could add an optional step for submitting the job with yaml, for debugging locally with minikube. In this way everyone can help debugging.